### ISSUE TITLE
Should be able to onboard & login using keyboard only

### ISSUE THREAD

**Issue by dexturr** [vegaprotocol/vegawallet-browser#230]
## Story

As a user
I want to be able to use all forms using my keyboard
So that the app is more accessible and easy to use

## Acceptance Criteria

- [ ] I can go through the onboarding flow using keyboard only
- [ ] I can login using keyboard only

## Tasks

- [ ] Detailed sketch
- [ ] Visual design
- [ ] Implement UI & Integrate with backend
- [ ] Write unit tests
- [ ] Write end to end tests
- [ ] Code review
- [ ] Team verify acceptance criteria met
- [ ] Exploratory QA testing 
- [ ] Capture bugs and new edge cases in tickets (this one or new TBD) 
- [ ] Write automated regression tests for the A/Cs (expanding on A/Cs where possible) 
- [ ] Add story and A/Cs to spec doc 
- [ ] Review with stakeholders
- [ ] Release
